index.js

```diff
@@ -64,7 +64,7 @@ exports.init = function (sbot, config) {
   var layer = sbot.friends.createLayer('user-invites')
 
   var caps = config.caps || {}
-  caps.userInvite = caps.userInvite || require('./cap')
```

Comment:
oh sorry, I'm gonna wreck this by merging a change which renames all `userInvites` to peerInvites` !!